src/vm/docs/mod.rs

```diff
@@ -84,6 +84,20 @@ contract principal.",
     example: "(print tx-sender) ;; Will print out a Stacks address of the transaction sender",
 };
 
+const TOTAL_LIQUID_USTX_KEYWORD: KeywordAPI = KeywordAPI {
+    name: "total-liquid-ustx",
```

Comment:
`total-liquid-ustx`, or `stx-get-liquid-supply`? Prefixing with `stx-` would be more consistent with our naming convention.

Reply:
It's a global variable, not a function, so I've avoided using a verb in the name (i.e. `get`).  In general, I'm happy to call it something else though.

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/1850/commits/87f39fa214cc8b7fb6d9f93b7440f26e792ab2cb